src/scripts/modules/media/body/body-template.html

```diff
@@ -16,5 +16,6 @@
         <span class="sr-only">Loading</span>
       </div>
     </div>
+    <div class="spacer"></div>
```

Comment:
my understand of it is to hint to the browser to save space for the header that is about to be rendered, this avoiding a flash/reflow. So, yes, a hack, sort of. Got a better way?